src/export-transactions.ts

```diff
@@ -121,7 +116,9 @@ async function getTransactions(name: string, tokenAddress: string, network: numb
         quote: trans.delta_quote,
         sender: trans.from_address,
         from: trans.from_address,
-        to: trans.to_address
+        to: trans.to_address,
+        interactedWith: tx.to_address,
```

Comment:
What is the difference between trans and tx? Maybe they can have different more explanatory names.